src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -59,11 +90,52 @@ const ContentHighlightsCardItemsContainer = ({
               onClick: () => trackClickEvent({ aggregationKey }),
             }
         }
-          contentType={contentType.toLowerCase()}
-          partners={authoringOrganizations}
-        />
-      ))}
-    </CardGrid>
+            contentType={contentType.toLowerCase()}
+            partners={authoringOrganizations}
+          />
+        ))}
+      </CardGrid>
+      {archivedContent.length > 0 && (
+        <>
+          <ActionRow>
+            <h3 className="m-0">
+              Archived
+            </h3>
+            <ActionRow.Spacer />
+            <Button variant="outline-primary">Delete archived courses</Button>
+          </ActionRow>
+          <div className="mb-4.5">Learners are no longer able to enroll in archived courses,
+            but past learners can still access course materials.
+          </div>
+          <CardGrid columnSizes={HIGHLIGHTS_CARD_GRID_COLUMN_SIZES}>
+            {archivedContent.map(({
+              uuid, title, contentType, authoringOrganizations, contentKey, cardImageUrl, aggregationKey,
+            }) => (
+              <ContentHighlightCardItem
+                isLoading={isLoading}
+                key={uuid}
+                cardImageUrl={cardImageUrl}
+                title={title}
+                archived
+                hyperlinkAttrs={
+          {
+            href: generateAboutPageUrl({
+              enterpriseSlug,
+              contentType: contentType.toLowerCase(),
+              contentKey,
+            }),
+            target: '_blank',
+            onClick: () => trackClickEvent({ aggregationKey }),
+          }
+      }
```

Comment:
Indentation for `hyperlinkAttrs` looks off.  Optionally might consider putting the whole `ContentHighlightCardItem` generation into a helper method to aid in readability.